### DISCUSSION TITLE
Sidecar destroyed due to temporary drive failure

### DISCUSSION THREAD

**Discussion by SinTan1729** [photoprism/photoprism#2744] [Q&A]
I had my HDD fail today, and before I could notice it, ofelia decided to run an indexing task, which obliterated all my index. The drive is working for now. I had a backup sql dump, so I was able to restore all my stuff, but currently I'm having to run a complete rescan of the library which takes hours and is simply torturing the already failing disk.

I'm replacing this disk ASAP but what should I do to avoid this in the future? Is it possible for ofelia to check if the directories are even present before proceeding to run an index? Syncthing deals with this kinda stuff by having a hidden file which works as a marker. If that's not possible, is it possible to implement a few steps of undoability for the indexing?

**Comment by graciousgrey** [comment]
If you are running the latest release the indexing should not start in case the originals folder is completely empty https://github.com/photoprism/photoprism/issues/2299

Of course, that doesn't help if only part of your files are missing.

> I'm replacing this disk ASAP but what should I do to avoid this in the future?

 To be 100% sure remove the auto index job until you have a new disk
